docs/rfc/templates/000-template.md

```diff
@@ -21,6 +21,28 @@
 > alternative solutions. In other words, enumerate the constraints you are trying to solve without coupling them too
 > closely to the solution you have in mind.
 
+## Requirements
+
+> What specific requirements does your design need to meet? This should ideally be a bulleted list of items you wish
+> to achieve with your design. This can help everyone involved (including yourself!) make sure your design is robust
+> enough to meet these requirements.
+>
+> Once everyone has agreed upon the set of requirements for your design, we can use this list to review the detailed
+> design.
+
+### Extensibility
```

Comment:
Why is this a sub-section of requirements? Shouldn't it be top level or a sub-section of detailed design?

Reply:
Extensibility _requirements_? I'm fine either way, but this a little clearer is it still a list of requirements.

Reply:
Would be good to have this as a separate section as part of "Future Work".